homeassistant/components/proxmoxve/__init__.py

```diff
@@ -186,6 +175,73 @@ def build_client() -> ProxmoxAPI:
     return True
 
 
+async def get_nodes(hass, config):
+    """Get Proxmox cluster nodes."""
+    hass.data[PROXMOX_HOSTS] = {}
+
+    def get_nodes_call():
+        nodes = proxmox.nodes.get()
+        return nodes
+
+    for entry in config[DOMAIN]:
```

Comment:
The config is already passed into `async def get_nodes(hass, config)`, and we already iterating the host?

Why are we looking for the host again now?